tce/CHANGES

```diff
@@ -36,6 +36,7 @@ Usability features
 - List filtering in ProDe dialogs 'Add from opset' and 'Add FU from HDB'.
 - Directory where ttasim dumps simulation traces can be changed using an
   environment variable TTASIM_TRACE_DIR.
+- Added source code debugging window to Proxim.
```

Comment:
We can call this a major new feature as it has been requested for so long time and is so useful.